### PR TITLE
Check upgrade

### PR DESCRIPTION
**Pull request by madhuBayy**
@ldko @somexpert 
Missed renaming `test_unicode` with `test_str` during python3 upgrade, so fixed it. And changed python base image in Dockerfile. Please review the changes.

### CODE REVIEW

**Review by ldko — COMMENTED** [review]
Do we also need to add a migration?

**Comment by madhuBayy** [comment]
I don't think so 'cos I have renamed the functions in tests and changed the base image of Docker container.

**Comment by ldko** [comment]
I mean in regard to changes made previously on py3 branch. When I deploy this in Docker and run migrations it says "Your models have changes that are not yet reflected in a migration."

**Comment by madhuBayy** [comment]
oh...Let me try it.
EDIT: I'm gonna push the migrations.

**Review by ldko — APPROVED** [review]
Works for me :+1:

File: accession/migrations/0002_auto_20191031_2139.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        ),
+        migrations.AlterField(
+            model_name='object',
+            name='label',
+            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, to='accession.Label', verbose_name='Designer Label'),
+        ),
+        migrations.AlterField(
+            model_name='object',
+            name='object_era',
+            field=models.CharField(blank=True, choices=[('Pre-1770', 'Pre-1770'), ('1770-1779', '1770-1779'), ('1780-1789', '1780-1789'), ('1790-1799', '1790-1799'), ('1800-1809', '1800-1809'), ('1810-1819', '1810-1819'), ('1820-1829', '1820-1829'), ('1830-1839', '1830-1839'), ('1840-1849', '1840-1849'), ('1850-1859', '1850-1859'), ('1860-1869', '1860-1869'), ('1870-1879', '1870-1879'), ('1880-1889', '1880-1889'), ('1890-1899', '1890-1899'), ('1900-1909', '1900-1909'), ('1910-1919', '1910-1919'), ('1920-1929', '1920-1929'), ('1930-1939', '1930-1939'), ('1940-1949', '1940-1949'), ('1950-1959', '1950-1959'), ('1960-1969', '1960-1969'), ('1970-1979', '1970-1979'), ('1980-1989', '1980-1989'), ('1990-1999', '1990-1999'), ('2000-2009', '2000-2009'), ('2010-2019', '2010-2019')], max_length=10),
```
Comment (somexpert) [inline accession/migrations/0002_auto_20191031_2139.py]: Just happened to notice that we stop at 2019, which makes sense, but we're almost to 2020. @ldko, do you think we should go ahead and add the next decade now in the model and then replace this migration with another one after that change?

**Comment by ldko** [comment]
@somexpert that makes sense to me based on the way eras are being handled.

**Comment by somexpert** [comment]
@madhulika95b, I think it would be better to just have one new migration that includes all the changes, so can you manually delete the two migrations and then create one new one for this MR?

**Review by somexpert — APPROVED** [review]
:thumbsup:

**Comment by ldko** [comment]
:+1: the migrations look good do me

**Pull request merged**